packages/runtime/src/ctx.ts

```diff
@@ -382,6 +404,10 @@ export class Ctx implements Context {
 			throw new RuntimeError(RUNTIME_ERRORS.GENERAL.INVALID_CLEAR_PROGRAM);
 		}
 
+		const approvalProgramBytes = Buffer.from(approvalProgTEAL, "base64");
+		const clearProgramBytes = Buffer.from(clearProgTEAL, "base64");
```

Comment:
the bytes program should be encoded from [algodClient](https://algorand.github.io/js-algorand-sdk/classes/Algodv2.html#compile) 
We also discuss about this one but have not found any idea yet [Discord message](https://discord.com/channels/754286650540032030/754286650540032033/1025398047649308734)

Reply:
Done